cvat/apps/engine/serializers.py

```diff
@@ -924,7 +924,7 @@ class FrameMetaSerializer(serializers.Serializer):
     width = serializers.IntegerField()
     height = serializers.IntegerField()
     name = serializers.CharField(max_length=1024)
-    has_related_context = serializers.BooleanField()
+    related_files = serializers.IntegerField()
```

Comment:
I would prefer to keep here the list of names. But let's keep it as is for now. Need to think twice.